app/controllers/plans_controller.rb

```diff
@@ -1,3 +1,14 @@
+# Handle the plans model and the worksheet.
+#
+# Plans are typically created from the GoalsController, but are displayed,
+# edited, deleted, etc. from here. This is unusual for a REST app, but plans
+# are not really created from a plan sheet itself, but from the concept of a
+# set of Goals.
+#
+# Otherwise, updates, deletes, and shows work as expected.
+#
+# The Index operation (GET /plans/) displays the list of plans owned by the
+# user, which is a different template.
```

Comment:
"which is a different template" is a bit ambiguous. we could say that the index action shows a list of plans and  populates country and assessment drop down menus

Reply:
Decided to just drop the "which is a different template" text since it doesn't add much.